README.md

```diff
@@ -233,6 +234,56 @@ Check the  [Notification Configuration](#37-notification-configuration) to see h
 
 For more information, please check the [Global Setting Configuration](#38-global-setting-configuration)
 
+
+### 1.4 Administration
+
+There are some administration configuration options:
+
+**1) PID file**
+
+  The EaseProbe would create a PID file (default `$CWD/easeprobe.pid`) when it starts. it can be configured by:
+
+  ```YAML
+  settings:
+    pid: /var/run/easeprobe.pid
+  ```
+
+  - If the file already exists, EaseProbe would overwrite it. 
+  - If the file cannot be written, EaseProbe would exit with an error.
+
+  If you want to disable the PID file, you can configure the pid file to "".
+
+  ```YAML
+  settings:
+    pid: "" # EaseProbe won't create a PID file
+  ```
+
+**2) Log file Rotation**
+
+  There are two types of log file: **Application Log** and **HTTP Access Log**. 
+
+  Both Application Log and HTTP Access Log would be StdOut by default.  They all can be configured by:
+
+  ```YAML
+  log:
+    file: /path/to/log/file
+    self_rotate: true # default: true
+  ```
+
+  If `self_rotate` is `true`, EaseProbe would rotate the log automatically, and the following options are available: 
+
+  ```YAML
+    size: 10 # max size of log file. default: 10M
+    age: 7 # max age days of log file. default: 7 days
+    backups: 5 # max backup log files. default: 5
+    compress: true # compress. default: true
+  ```
+
+  If `self_rotate` is `false`, EaseProbe would not rotate the log, and the log file would be rotated by the 3rd-party tool. e.g. `logrotate`.
```

Comment:
and the log file would have to be rotated by a 3rd-party tool (such as `logrotate`) or manualy by the administrator
```shell
mv /path/to/easeprobe.log /path/to/easeprobe.log.0 
kill -HUP `cat /path/to/easeprobe.pid`
```

Reply:
The document updated!